language/resource.language.it_it/strings.po

```diff
@@ -4258,13 +4258,13 @@ msgstr ""
 #: /1080i/Includes_Labels.xml
 msgctxt "#31874"
 msgid "total"
-msgstr ""
+msgstr "Totali"
```

Comment:
Can this be changed to "totali", this word is used at the end of an awards sentence.